packages/mason_cli/test/fixtures/unbundle/brick.yaml

```diff
@@ -0,0 +1 @@
+{"name":"greeting","description":"A Simple Greeting Template","version":"0.1.0+1","environment":{"mason":"any"},"vars":{"name":{"type":"string","description":"Your name","default":"Dash","prompt":"What is your name?"}}}
```

Comment:
You probably should remove the newline here since it’ll likely cause windows and unix implementations to differ.

Reply:
I am unable to do it since I am working from Windows PC atm and no line break is displayed 😕

Reply:
No worries I can help with that if needed 👍